brainstormforce/ultimate-addons-for-gutenberg
# Fix: Marketing Button - issues reported by QA in next-release.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/4a6816bd-1ea2-4ecd-9b55-5fdc995e262a?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Check default value matches the same as the frontend as well as in the editor for Icon size.
- Check background > gradient working properly on front.
- Check default border width is set or not.

### Checklist:
